.github/workflows/review-code.yaml

```diff
@@ -51,7 +51,7 @@ jobs:
             file_content=$(git show "${{ github.event.pull_request.head.sha }}:$file" | jq -Rsa .)
 
             # Send the diff and file content to the backend for each file separately
-            curl -X POST "https://app5-latest.onrender.com/openai/api/v2/strategy/4/review-pr-diff" \
+            curl -X POST "https://app5-latest.onrender.com/ai/api/v1/strategy/1/review-pr-diff" \
```

Comment:
The use of a hardcoded URL can lead to security vulnerabilities and makes the code less flexible. If the URL changes, it requires code modification. Additionally, this line does not conform to Java syntax and should be part of a Java method. (Severity: high)